dive_core/command_hierarchy.cpp

```diff
@@ -554,7 +554,9 @@ bool CommandHierarchyCreator::CreateTrees(CommandHierarchy  *command_hierarchy_p
     }
 
     m_num_events = 0;
+    m_cur_shared_node_parent_index = UINT64_MAX;
```

Comment:
probably a good candidate for a Reset/Clear function since we have these reset/clear code duplicated 3 times in this file